safekeeper/src/safekeeper.rs

```diff
@@ -193,9 +194,9 @@ pub struct SafeKeeperState {
     /// Part of WAL acknowledged by quorum and available locally. Always points
     /// to record boundary.
     pub commit_lsn: Lsn,
-    /// First LSN not yet offloaded to s3. Useful to persist to avoid finding
-    /// out offloading progress on boot.
-    pub s3_wal_lsn: Lsn,
+    /// LSN that points to the end of the last backed up segment. Useful to
+    /// persist to avoid finding out offloading progress on boot.
+    pub backup_lsn: Lsn,
```

Comment:
Should we bump control file version and add migration for it, or this simple rename will work as is?

Reply:
Tested, it works. Apparently bincode doesn't put field name inside.